src/tools/auth0/handlers/tenant.ts

```diff
@@ -1,16 +1,16 @@
+import { TenantSettings, TenantSettingsFlags, TenantSettingsUpdate, TenantSettingsUpdateFlags } from 'auth0';
 import ValidationError from '../../validationError';
-
 import DefaultHandler, { order } from './default';
 import { supportedPages, pageNameMap } from './pages';
 import { convertJsonToString } from '../../utils';
-import { Asset, Assets, Language } from '../../../types';
+import { Asset, Assets } from '../../../types';
 import log from '../../../logger';
 
 export const schema = {
   type: 'object',
 };
 
-export type Tenant = Asset & { enabled_locales?: Language[]; flags: { [key: string]: boolean } };
+export type Tenant = TenantSettings;
```

Comment:
Would rather we just get rid of the `Tenant` type altogether in favor of the `TenantSettings`.